ui/apps/platform/cypress/integration/vulnerabilities/nodeCves/cveDetailPage.test.ts

```diff
@@ -0,0 +1,73 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+
+describe('Node CVEs - CVE Detail Page', () => {
+    withAuth();
+
+    before(function () {
+        if (!hasFeatureFlag('ROX_VULN_MGMT_NODE_PLATFORM_CVES')) {
+            this.skip();
+        }
+    });
+
+    it('should restrict access to users with insufficient permissions', () => {
+        // check that users without Node access cannot access the Node CVE Detail page directly
+    });
+
+    it('should only show relevant filters for the Node CVE Detail page', () => {
+        // check the advanced filters and ensure only the relevant filters are displayed
+    });
+
+    it('should link to the correct pages', () => {
+        // clicking the Node CVEs breadcrumb should navigate to the overview page with the CVE tab selected
+        // clicking a Node name in the list should navigate to the correct Node details page
+    });
+
+    it('should sort Node table columns', () => {
```

Comment:
What do you think about a separate and simpler test for presence of table columns, in case it becomes practical to test only a subset of the sorting?